pyscriptjs/src/components/pyconfig.ts

```diff
@@ -1,127 +1,18 @@
 import * as jsyaml from 'js-yaml';
 import { BaseEvalElement } from './base';
-import {
-    initializers,
-    loadedEnvironments,
-    postInitializers,
-    pyodideLoaded,
-    scriptsQueue,
-    globalLoader,
-    appConfig,
-    Initializer,
-} from '../stores';
-import { loadInterpreter } from '../interpreter';
-import type { PyLoader } from './pyloader';
-import type { PyScript } from './pyscript';
-import type { PyodideInterface } from '../pyodide';
+import { appConfig } from '../stores';
+import type { Runtime, AppConfig } from '../runtime';
+import { PyodideRuntime } from '../pyodide';
 
-const DEFAULT_RUNTIME = {
-    src: 'https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js',
-    name: 'pyodide-default',
-    lang: 'python',
-};
+/*
+Usage of initializers, postInitializers, scriptsQueue, etc.
```

Comment:
This also seems more like a PR comment rather than a code comment. 

Here you should describe in a more generic way "what's going on" so that people reading the code doesn't have to guess (without context)

Reply:
done, thanks!